### PR TITLE
Edit destroy worksession

### PR DESCRIPTION
**Pull request by Alexisreau97**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,5 +4,5 @@
   resources :teachers, only: %i[index]
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :students, only: %i[index edit destroy update]
-  resources :work_sessions, only: %i[new create index], path: 'sessions'
+  resources :work_sessions, only: %i[new create index edit update destroy], path: 'sessions'
```
Comment (Crysicia) [inline config/routes.rb]: Useless `only` here, you are using a full CRUD there

File: app/controllers/work_sessions_controller.rb [hunk 2]
```diff
@@ -18,15 +20,42 @@ def index
     @work_sessions = WorkSession.all.order(:date)
   end
 
+  def edit
+    @skills = Skill.all
+    @slots = Slot.all
+    @teachers = Teacher.all
+    @students = Student.all
```
Comment (Crysicia) [inline app/controllers/work_sessions_controller.rb]: Could use the before action implemented in the last PR

**Review by Crysicia — CHANGES_REQUESTED** [review]

File: app/views/students/_table.html.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      <th>Nom</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @students.each do |student| %>
+      <tr>
+        <td><%= student.first_name %></td>
+        <td><%= student.last_name.upcase %></td>
+        <td><%= link_to "Éditer", edit_student_path(student.id) %></td>
+        <td><%= link_to 'Delete', student_path(student.id), method: :delete, data: { confirm: 'Êtes-vous sûr ?' } %></td>
```
Comment (Crysicia) [inline app/views/students/_table.html.erb]: You can just put `student` instead of `student.id`

File: app/views/work_sessions/_table.html.erb [hunk 4]
```diff
@@ -10,6 +14,8 @@
     <tr>
       <td><%=work_session.date.strftime("%d %b %Y at %l:%M")%></td>
       <td><%=work_session.title%></td>
+      <td><%= link_to "Éditer", edit_work_session_path(work_session.id) %></td>
+      <td><%= link_to 'Delete', work_session_path(work_session.id), method: :delete, data: { confirm: 'Êtes-vous sûr ?' } %></td>
```
Comment (Crysicia) [inline app/views/work_sessions/_table.html.erb]: Again, you can just put `work_session` instead of `work_session.id`

**Comment by Crysicia** [comment]
It may be cleaner to put the form in a partial, and then call it for `new` and `edit`

**Review by orohellec — APPROVED** [review]

**Review by Crysicia — APPROVED** [review]

**Pull request merged**